### PR TITLE
Tweak layout of services to conveniently handle large service families [appveyor cdnjs clojars gem npm]

### PR DESCRIPTION
**Pull request by paulmelnikow**
As discussed here:
https://github.com/badges/shields/pull/1743#issue-195899374
https://github.com/badges/shields/pull/1743#issuecomment-398901078

Todo:
- [x] Adjust danger rules

### CODE REVIEW

File: services/index.js [hunk 1]
```diff
@@ -2,14 +2,9 @@
...
 
-// Match modules with the same name as their containing directory.
-// e.g. services/appveyor/appveyor.js
-const serviceRegex = /\/services\/(.*)\/\1\.js$/;
-
 function loadServiceClasses() {
   // New-style services
-  const services = glob.sync(`${__dirname}/**/*.js`)
-    .filter(path => serviceRegex.test(path))
+  const services = glob.sync(`${__dirname}/**/*.service.js`)
```
Comment (chris48s) [inline services/index.js]: Should this still be filtered to only files under the `/services` directory? If we end up with a file called `something.service.js` in `/lib` or something I think we probably don't want to pull it in here

**Comment by chris48s** [comment]
@RedSparr0w note that if this gets merged ahead of #1785 and/or #1802 they'll need rebasing to account for this, even though there won't be a merge conflict

or possibly the other way round if one or other of those goes in ahead of this.

File: services/index.js (hunk 1 above)
Comment (paulmelnikow) [reply services/index.js]: Yea, true, though this is `services/index.js` so `__dirname` here is `/services`.

Comment (chris48s) [reply services/index.js]: Oh yes, good point. Looks like that call to `.filter()` was not doing anything useful :)

Comment (paulmelnikow) [reply services/index.js]: Before it was filtering out the testers, and any random helper modules that might have been stashed in `services/`. 😉

**Comment by paulmelnikow** [comment]
Hey, could I get your 👍's on this?

**Comment by chris48s** [comment]
> Hey, could I get your +1's on this?

The changes to the service layout itself are fine, but there's still an issue with the changes to `dangerfile.js`. If I run `danger` against this PR using the config on `master` it runs fine, but if I checkout this PR I get the same failure as show in https://github.com/badges/shields/pull/1797#issuecomment-406925979 locally.

**Comment by paulmelnikow** [comment]
Should be fixed, thanks!

**Pull request merged**